Quaver.Shared/Screens/Select/SelectScreen.cs

```diff
@@ -542,5 +580,177 @@ public void SelectRandomMap()
                     break;
             }
         }
+
+        /// <summary>
+        ///     Used to start the deleting process.
+        ///
+        ///     woo now AiAe will stop telling me to add this.
+        /// </summary>
+        /// <param name="type"></param>
+        private void DeleteSelected()
```

Comment:
Breaks when trying to delete a mapset when its the only one loaded.

Reply:
Deleting a difficulty doesn't update the map banner.

Reply:
Deleting a mapset when its the only one loaded and then trying to import another results in a thrown exception